packages/core/src/user-store/utils/pagination.ts

```diff
@@ -1,8 +1,12 @@
+import { createGraphQLError } from "graphql-yoga";
+
 const MAX_SKIP = 5000;
 
 export function validateSkip(skip: number) {
   if (skip > MAX_SKIP) {
-    throw new Error("Invalid query. Cannot skip more than 5000 rows.");
+    throw createGraphQLError(
```

Comment:
this fixes this error:

https://github.com/0xOlias/ponder/actions/runs/6387778466/job/17336588381?pr=367#step:5:54

Reply:
got this from here: https://github.com/dotansimha/graphql-yoga/blob/0f1483c939ccb76f5e3c809c0489fa6cc9aba864/examples/error-handling/src/yoga.ts#L43-L51

Without it, the error doesn't include the message. The error message is [`"Unexpected error."`](https://github.com/0xOlias/ponder/actions/runs/6387778466/job/17336588381?pr=367#step:5:116)